README.md

```diff
@@ -103,6 +103,9 @@ S3 service (minio), DB and will use the image generator to create the images in
 skipper make deploy-all
 ```
 
+Note: when deployed in minikube, we need to be running [minikube tunnel](https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-tunnel), as some services are type `LoadBalancer` and need to be exposed.
+
```

Comment:
might be useful to mention this command for readers to know it to run it locally in the background:
```sh
nohup minikube tunnel &>/dev/null
```